packages/calcite-components/src/components/action-bar/readme.md

```diff
@@ -4,6 +4,10 @@
 
 The `calcite-action-bar` component is made up of multiple `calcite-action`s in the form of clickable icons. The action bar can be expanded to view actions with descriptive text or made smaller to view with just icons.
 
+Action bars should be placed on the left or right edge of the interface. For a floating group of actions, please consider Action Pad.
```

Comment:
I don't think we need either of these additional items. Action bars can also exist at the top / bottom of a UI or within Panel or other elements in any configuration.